canonical-web-and-design/kubeflow-news.com
# Add the @canonical modules to the site

Question: @pmahnke do you want the global nav on this site? Happy to remove if not.

## Done
Add the global-nav and the cookie policy modules

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8035/
- Check that the global nav works
- Make a selection in the cookie policy and see its gone
- Scroll to the bottom and click the Manage link
- See it brings back the cookie policy

## Issue / Card
Fixes https://github.com/canonical-web-and-design/kubeflow-news.com/issues/69
